auth/tests/api/auth.rs

```diff
@@ -5,64 +5,89 @@ mod needs_docker {
     use serde_json::Value;
     use shuttle_common::claims::AccountTier;
 
-    use crate::helpers::{app, ADMIN_KEY};
+    use crate::helpers::app;
 
     #[tokio::test]
     async fn convert_api_key_to_jwt() {
         let app = app().await;
 
-        // Create test user
-        let response = app.post_user("test-user", "basic").await;
-
+        // Create test basic user
+        let response = app.post_user("test-user-basic", "basic").await;
         assert_eq!(response.status(), StatusCode::OK);
+        // Extract the API key from the response so we can use it in a future request.
+        let body = hyper::body::to_bytes(response.into_body()).await.unwrap();
+        let user: Value = serde_json::from_slice(&body).unwrap();
+        let basic_user_key = user["key"].as_str().unwrap();
 
+        // Create test admin user
+        let response = app.post_user("test-user-admin", "admin").await;
```

Comment:
We create an admin user when we initialize the app, for which the API key is available as a constant, could we use that to make the test simpler?

Reply:
Yes, definitely. Thanks for the suggestion, done!